aop/extensions/diff.py

```diff
@@ -0,0 +1,31 @@
+import os
+import json
+from distutils.util import strtobool
+from aop.aop import extends
+from editor import diff as df
+from flask import request
+
+feature_diff = bool(strtobool(os.getenv("USE_DIFF")))
+
+if feature_diff:
+    print("Using feature: Diff")
+
+    @extends('define_endpoints')
+    def diff_endpoints(flaskapp):
+        @flaskapp.route('/editor/diff', methods=['POST'])
+        def diff_files():
+            incoming = request.get_json()
+            html = df.diffOnSentences(incoming["one"], incoming["two"])
+            return html, 200
```

Comment:
Maybe do this with `jsonify`, so you'd get something more akin to `return jsonify(success=False, message='msg: {}'.format(message))`